tests/common-utils.js

```diff
@@ -140,6 +140,7 @@ commonUtils.asyncLoadScript = function (url) {
         script.onload();
       }
     };
+    script.onerror = reject;
```

Comment:
Not directly relevant to this PR.  This needs testing separately, and including as its own PR if it works.